pkg/interface/src/views/apps/chat/components/ChatWindow.tsx

```diff
@@ -167,8 +172,10 @@ export default class ChatWindow extends Component<ChatWindowProps, ChatWindowSta
   }
 
   scrollToUnread() {
-    const { mailboxSize, unreadCount } = this.props;
-    this.virtualList?.scrollToData(mailboxSize - unreadCount);
+    const { mailboxSize, unreadCount, scrollTo } = this.props;
+    const target = scrollTo || (mailboxSize - unreadCount);
+    console.log('scrolling', target);
```

Comment:
oy

Reply:
that's it, i'm adding a pre-commit hook to check for `console.log`

Reply:
must-have